### PR TITLE
Changed the default opacity to 0.7

### LINKED ISSUE #361: Default opacity would be better off with values such as 0.6 or 0.75
Currently the default opacity is 1.0 which hides the basemap. I think it is common to have default opacity set to 0.6 or something close so that we can see the basemap.

### PR DESCRIPTION
**Pull request by dorukozturk**
Fixes #361

### CODE REVIEW

File: web_external/js/models/DatasetModel.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ minerva.models.DatasetModel = minerva.models.MinervaModel.extend({
      */
     initialize: function () {
         minerva.models.MinervaModel.prototype.initialize.apply(this, arguments);
-        this.on('change:geoData', this._preprocess, this);
+        this.on('change:geoData', this._preprocesps, this);
```
Comment (jbeezley) [inline web_external/js/models/DatasetModel.js]: Typo?

File: web_external/js/views/adapters/Adapters.js [hunk 2]
```diff
@@ -304,7 +304,8 @@ minerva.rendering.geo.WmsRepresentation = minerva.rendering.geo.defineMapLayer('
     this.init = function (container, dataset) {
         this.geoJsLayer = container.createLayer('osm', {
             attribution: null,
-            keepLower: false
+            keepLower: false,
+            opacity: dataset.get('opacity')
```
Comment (jbeezley) [inline web_external/js/views/adapters/Adapters.js]: You should probably make this change to the other representations as well.

**Comment by aashish24** [comment]
@dorukozturk can you update this branch?

**Comment by aashish24** [comment]
@dorukozturk should we close this PR?

**Pull request closed without merging**